README.md

```diff
@@ -1,24 +1,77 @@
-[![tests](https://github.com/ddev/ddev-redis/actions/workflows/tests.yml/badge.svg)](https://github.com/ddev/ddev-redis/actions/workflows/tests.yml) ![project is maintained](https://img.shields.io/maintenance/yes/2024.svg)
+<div align="center">
 
-## What is this?
+# ddev-redis - Redis container for DDEV
 
-This repository allows you to quickly install Redis into a [DDEV](https://ddev.readthedocs.io) project using just `ddev get ddev/ddev-redis`.
+[![GitHub release (with filter)](https://img.shields.io/github/v/release/ddev/ddev-redis)](https://github.com/ddev/ddev-redis/releases)
+[![Tests](https://github.com/ddev/ddev-redis/actions/workflows/cron_tests.yml/badge.svg)](https://github.com/ddev/ddev-redis/actions/workflows/cron_tests.yml)
+![project is maintained](https://img.shields.io/maintenance/yes/2024.svg)
+
+</div>
+
+This repository provides [Redis](https://redis.com) container for [DDEV](https://ddev.readthedocs.io/).
+
+It is based on [redis:6-alpine](https://hub.docker.com/layers/library/redis/6-alpine) docker image and [DDEV custom compose files](https://ddev.readthedocs.io/en/stable/users/extend/custom-compose-files/)
+
+## Version 2 features
+
+There are a lot of differences between [official](https://github.com/ddev/ddev-redis) addon and this one
+
+| Feature           | v1               | v2                       |
+| ----------------- | ---------------- | ------------------------ |
+| Maximum Memory    | Unlimited        | 512Mb                    |
+| Persistence       | No               | **Yes**                  |
+| Image Size        | 112.9Mb          | 14.38Mb                  |
```

Comment:
In fact, these are not fair numbers.

14.38Mb - compressed `7-alpine`
112.9Mb - uncompressed `6-bullseye`

Reply:
![image](https://github.com/ddev/ddev-redis/assets/6689756/e8ee4a44-ccda-4448-bdb2-9f66efce1123)

Can we agree on 30 / 113? :)